tests/BaseOAuthTest.php

```diff
@@ -44,7 +58,7 @@ public function testSetGet()
     }
 
     /**
-     * @runInSeparateProcess
+     * @ runInSeparateProcess
```

Comment:
Is it a test?

Reply:
Yes, but doesn’t work with this annotation.

Reply:
1. It is preferred that tests work without `@runInSeparateProcess`.
2. It is preferred that tests are OK when executed in random order or one by one.

Reply:
Fixed